src/stellar/stellarEngine.js

```diff
@@ -242,7 +242,16 @@ export class StellarEngine extends CurrencyEngine {
           pagingToken = await this.processTransaction(tx)
         }
       } catch (e) {
-        pagingToken = undefined
+        if (e.response && e.response.title === 'Resource Missing') {
+          this.log('Account not found. Probably not activated w/minimum XLM')
```

Comment:
Only QA / support sees these logs. I presume this is for their benefit. Does the core need a way to surface this through to the GUI as a proper error / plugin status flag?